src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -168,6 +174,17 @@ public override async Task<IActionResult> GetBuildGraph(int id)
             return Ok(new Models.Build(build));
         }
 
+        [HttpGet("{buildId}/commit")]
+        [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(Build), Description = "The commit matching specified criteria")]
+        [ValidateModelState]
+        public async Task<IActionResult> GetCommit(int buildId)
+        {
+            Data.Models.Build build = await _context.Builds.Include(b => b.Incoherencies).FirstOrDefaultAsync(b => b.Id == buildId);
```

Comment:
You call "OrDefault" but then reference it on the next line.  Calling "OrDefault", but then failing to handle the null case is strange.

There should be an return NotFound in here somewhere if we can't find the build you suggested.